[2024-07-22T13:46:41.620+0000] {processor.py:153} INFO - Started process (PID=1737) to work on /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:46:41.620+0000] {processor.py:640} INFO - Processing file /opt/airflow/dags/P2M3_michael_DAG.py for tasks to queue
[2024-07-22T13:46:41.620+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:46:41.620+0000] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:46:41.625+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:46:41.624+0000] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/P2M3_michael_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/P2M3_michael_DAG.py", line 58
    'retries': 1,
            ^
SyntaxError: invalid syntax
[2024-07-22T13:46:41.625+0000] {processor.py:652} WARNING - No viable dags retrieved from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:46:41.717+0000] {processor.py:161} INFO - Processing /opt/airflow/dags/P2M3_michael_DAG.py took 0.099 seconds
[2024-07-22T13:47:11.979+0000] {processor.py:153} INFO - Started process (PID=1793) to work on /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:47:11.980+0000] {processor.py:640} INFO - Processing file /opt/airflow/dags/P2M3_michael_DAG.py for tasks to queue
[2024-07-22T13:47:11.980+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:47:11.980+0000] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:47:11.985+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:47:11.984+0000] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/P2M3_michael_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/P2M3_michael_DAG.py", line 58
    'retries': 1,
            ^
SyntaxError: invalid syntax
[2024-07-22T13:47:11.985+0000] {processor.py:652} WARNING - No viable dags retrieved from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:47:12.077+0000] {processor.py:161} INFO - Processing /opt/airflow/dags/P2M3_michael_DAG.py took 0.099 seconds
[2024-07-22T13:47:42.415+0000] {processor.py:153} INFO - Started process (PID=1849) to work on /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:47:42.415+0000] {processor.py:640} INFO - Processing file /opt/airflow/dags/P2M3_michael_DAG.py for tasks to queue
[2024-07-22T13:47:42.416+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:47:42.416+0000] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:47:42.420+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:47:42.419+0000] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/P2M3_michael_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/P2M3_michael_DAG.py", line 58
    'retries': 1,
            ^
SyntaxError: invalid syntax
[2024-07-22T13:47:42.420+0000] {processor.py:652} WARNING - No viable dags retrieved from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:47:42.514+0000] {processor.py:161} INFO - Processing /opt/airflow/dags/P2M3_michael_DAG.py took 0.101 seconds
[2024-07-22T13:48:12.748+0000] {processor.py:153} INFO - Started process (PID=1905) to work on /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:48:12.748+0000] {processor.py:640} INFO - Processing file /opt/airflow/dags/P2M3_michael_DAG.py for tasks to queue
[2024-07-22T13:48:12.749+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:48:12.749+0000] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:48:12.759+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:48:12.758+0000] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/P2M3_michael_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/P2M3_michael_DAG.py", line 58
    'retries': 1,
            ^
SyntaxError: invalid syntax
[2024-07-22T13:48:12.759+0000] {processor.py:652} WARNING - No viable dags retrieved from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:48:12.845+0000] {processor.py:161} INFO - Processing /opt/airflow/dags/P2M3_michael_DAG.py took 0.099 seconds
[2024-07-22T13:48:43.295+0000] {processor.py:153} INFO - Started process (PID=1961) to work on /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:48:43.296+0000] {processor.py:640} INFO - Processing file /opt/airflow/dags/P2M3_michael_DAG.py for tasks to queue
[2024-07-22T13:48:43.296+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:48:43.296+0000] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:48:43.301+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:48:43.300+0000] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/P2M3_michael_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/P2M3_michael_DAG.py", line 58
    'retries': 1,
            ^
SyntaxError: invalid syntax
[2024-07-22T13:48:43.301+0000] {processor.py:652} WARNING - No viable dags retrieved from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:48:43.392+0000] {processor.py:161} INFO - Processing /opt/airflow/dags/P2M3_michael_DAG.py took 0.099 seconds
[2024-07-22T13:49:13.671+0000] {processor.py:153} INFO - Started process (PID=2017) to work on /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:49:13.671+0000] {processor.py:640} INFO - Processing file /opt/airflow/dags/P2M3_michael_DAG.py for tasks to queue
[2024-07-22T13:49:13.672+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:49:13.671+0000] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:49:13.676+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:49:13.675+0000] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/P2M3_michael_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/P2M3_michael_DAG.py", line 58
    'retries': 1,
            ^
SyntaxError: invalid syntax
[2024-07-22T13:49:13.676+0000] {processor.py:652} WARNING - No viable dags retrieved from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:49:13.766+0000] {processor.py:161} INFO - Processing /opt/airflow/dags/P2M3_michael_DAG.py took 0.097 seconds
[2024-07-22T13:49:43.979+0000] {processor.py:153} INFO - Started process (PID=2072) to work on /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:49:43.979+0000] {processor.py:640} INFO - Processing file /opt/airflow/dags/P2M3_michael_DAG.py for tasks to queue
[2024-07-22T13:49:43.979+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:49:43.979+0000] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:49:43.984+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:49:43.983+0000] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/P2M3_michael_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/P2M3_michael_DAG.py", line 58
    'retries': 1,
            ^
SyntaxError: invalid syntax
[2024-07-22T13:49:43.984+0000] {processor.py:652} WARNING - No viable dags retrieved from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:49:44.069+0000] {processor.py:161} INFO - Processing /opt/airflow/dags/P2M3_michael_DAG.py took 0.092 seconds
[2024-07-22T13:50:14.322+0000] {processor.py:153} INFO - Started process (PID=2128) to work on /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:50:14.322+0000] {processor.py:640} INFO - Processing file /opt/airflow/dags/P2M3_michael_DAG.py for tasks to queue
[2024-07-22T13:50:14.322+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:50:14.322+0000] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:50:14.327+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:50:14.326+0000] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/P2M3_michael_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/P2M3_michael_DAG.py", line 58
    'retries': 1,
            ^
SyntaxError: invalid syntax
[2024-07-22T13:50:14.327+0000] {processor.py:652} WARNING - No viable dags retrieved from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:50:14.411+0000] {processor.py:161} INFO - Processing /opt/airflow/dags/P2M3_michael_DAG.py took 0.091 seconds
[2024-07-22T13:50:44.786+0000] {processor.py:153} INFO - Started process (PID=2184) to work on /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:50:44.787+0000] {processor.py:640} INFO - Processing file /opt/airflow/dags/P2M3_michael_DAG.py for tasks to queue
[2024-07-22T13:50:44.787+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:50:44.787+0000] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:50:44.792+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:50:44.792+0000] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/P2M3_michael_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/P2M3_michael_DAG.py", line 58
    'retries': 1,
            ^
SyntaxError: invalid syntax
[2024-07-22T13:50:44.792+0000] {processor.py:652} WARNING - No viable dags retrieved from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:50:44.873+0000] {processor.py:161} INFO - Processing /opt/airflow/dags/P2M3_michael_DAG.py took 0.089 seconds
[2024-07-22T13:51:15.276+0000] {processor.py:153} INFO - Started process (PID=2239) to work on /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:51:15.276+0000] {processor.py:640} INFO - Processing file /opt/airflow/dags/P2M3_michael_DAG.py for tasks to queue
[2024-07-22T13:51:15.277+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:51:15.277+0000] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:51:15.282+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:51:15.282+0000] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/P2M3_michael_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/P2M3_michael_DAG.py", line 58
    'retries': 1,
            ^
SyntaxError: invalid syntax
[2024-07-22T13:51:15.283+0000] {processor.py:652} WARNING - No viable dags retrieved from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:51:15.365+0000] {processor.py:161} INFO - Processing /opt/airflow/dags/P2M3_michael_DAG.py took 0.091 seconds
[2024-07-22T13:51:45.604+0000] {processor.py:153} INFO - Started process (PID=2295) to work on /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:51:45.604+0000] {processor.py:640} INFO - Processing file /opt/airflow/dags/P2M3_michael_DAG.py for tasks to queue
[2024-07-22T13:51:45.605+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:51:45.604+0000] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:51:45.612+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:51:45.611+0000] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/P2M3_michael_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/P2M3_michael_DAG.py", line 58
    'retries': 1,
            ^
SyntaxError: invalid syntax
[2024-07-22T13:51:45.612+0000] {processor.py:652} WARNING - No viable dags retrieved from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:51:45.623+0000] {processor.py:161} INFO - Processing /opt/airflow/dags/P2M3_michael_DAG.py took 0.021 seconds
[2024-07-22T13:52:15.781+0000] {processor.py:153} INFO - Started process (PID=2351) to work on /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:52:15.782+0000] {processor.py:640} INFO - Processing file /opt/airflow/dags/P2M3_michael_DAG.py for tasks to queue
[2024-07-22T13:52:15.782+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:52:15.782+0000] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:52:15.788+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:52:15.787+0000] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/P2M3_michael_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/P2M3_michael_DAG.py", line 58
    'retries': 1,
            ^
SyntaxError: invalid syntax
[2024-07-22T13:52:15.788+0000] {processor.py:652} WARNING - No viable dags retrieved from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:52:15.799+0000] {processor.py:161} INFO - Processing /opt/airflow/dags/P2M3_michael_DAG.py took 0.020 seconds
[2024-07-22T13:52:45.829+0000] {processor.py:153} INFO - Started process (PID=2407) to work on /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:52:45.831+0000] {processor.py:640} INFO - Processing file /opt/airflow/dags/P2M3_michael_DAG.py for tasks to queue
[2024-07-22T13:52:45.831+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:52:45.831+0000] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:52:45.836+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:52:45.836+0000] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/P2M3_michael_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/P2M3_michael_DAG.py", line 58
    'retries': 1,
            ^
SyntaxError: invalid syntax
[2024-07-22T13:52:45.836+0000] {processor.py:652} WARNING - No viable dags retrieved from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:52:45.847+0000] {processor.py:161} INFO - Processing /opt/airflow/dags/P2M3_michael_DAG.py took 0.020 seconds
[2024-07-22T13:53:16.052+0000] {processor.py:153} INFO - Started process (PID=2464) to work on /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:53:16.052+0000] {processor.py:640} INFO - Processing file /opt/airflow/dags/P2M3_michael_DAG.py for tasks to queue
[2024-07-22T13:53:16.053+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:53:16.053+0000] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:53:16.058+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:53:16.057+0000] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/P2M3_michael_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/P2M3_michael_DAG.py", line 58
    'retries': 1,
            ^
SyntaxError: invalid syntax
[2024-07-22T13:53:16.058+0000] {processor.py:652} WARNING - No viable dags retrieved from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:53:16.069+0000] {processor.py:161} INFO - Processing /opt/airflow/dags/P2M3_michael_DAG.py took 0.019 seconds
[2024-07-22T13:53:46.262+0000] {processor.py:153} INFO - Started process (PID=2520) to work on /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:53:46.262+0000] {processor.py:640} INFO - Processing file /opt/airflow/dags/P2M3_michael_DAG.py for tasks to queue
[2024-07-22T13:53:46.263+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:53:46.262+0000] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:53:46.268+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:53:46.268+0000] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/P2M3_michael_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/P2M3_michael_DAG.py", line 58
    'retries': 1,
            ^
SyntaxError: invalid syntax
[2024-07-22T13:53:46.268+0000] {processor.py:652} WARNING - No viable dags retrieved from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:53:46.279+0000] {processor.py:161} INFO - Processing /opt/airflow/dags/P2M3_michael_DAG.py took 0.019 seconds
[2024-07-22T13:54:16.421+0000] {processor.py:153} INFO - Started process (PID=2576) to work on /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:54:16.421+0000] {processor.py:640} INFO - Processing file /opt/airflow/dags/P2M3_michael_DAG.py for tasks to queue
[2024-07-22T13:54:16.422+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:54:16.422+0000] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:54:16.428+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:54:16.428+0000] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/P2M3_michael_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/P2M3_michael_DAG.py", line 58
    'retries': 1,
            ^
SyntaxError: invalid syntax
[2024-07-22T13:54:16.428+0000] {processor.py:652} WARNING - No viable dags retrieved from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:54:16.440+0000] {processor.py:161} INFO - Processing /opt/airflow/dags/P2M3_michael_DAG.py took 0.021 seconds
[2024-07-22T13:54:23.439+0000] {processor.py:153} INFO - Started process (PID=2584) to work on /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:54:23.439+0000] {processor.py:640} INFO - Processing file /opt/airflow/dags/P2M3_michael_DAG.py for tasks to queue
[2024-07-22T13:54:23.440+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:54:23.440+0000] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:54:23.452+0000] {logging_mixin.py:115} INFO - [2024-07-22T13:54:23.451+0000] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/P2M3_michael_DAG.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/P2M3_michael_DAG.py", line 58
    'retries': 1,
            ^
SyntaxError: invalid syntax
[2024-07-22T13:54:23.452+0000] {processor.py:652} WARNING - No viable dags retrieved from /opt/airflow/dags/P2M3_michael_DAG.py
[2024-07-22T13:54:23.464+0000] {processor.py:161} INFO - Processing /opt/airflow/dags/P2M3_michael_DAG.py took 0.027 seconds
